### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by andrushkin2** [ilya-kovshik/form-translate#25]
1. Button that opens some popup window should have aria attributes: `aria-haspopup`, `data-popup-id`, and `aria-controls`
2. A popup should have `id` attribute 
3. When popup is opened and the popup is modal, user should not be able to navigate by elements on the page behind the popup using `Tab` key.

<img width="1318" alt="Screenshot 2022-05-05 at 13 19 13" src="https://user-images.githubusercontent.com/3307543/166904856-aa8a86e1-23ec-42a8-9f61-af53884d119b.png">

**Comment by ilya-kovshik** [comment]
Fixes have been pushed - [commit](https://github.com/ilya-kovshik/form-translate/commit/2b97831d4deb2b8d191ca3b3b8f6d4792593fe54)
